manuscript/060_Statement_First_Programming.md

```diff
@@ -110,13 +110,13 @@ public void ShouldRecognizeTimeSlotAboveMaximumAllowedAsInvalid()
 }
 ```
 
```

Comment:
I'll finish reviewing this tomorrow. Thank you for your hard work. I have finishing another chapter on value objects in plans and then I'm getting back to this chapter to make it better. I agree with most of what I wrote, so I expect ca. 95% reuse, but as it is, it's written in an offensive "in-your-face" way that I thankfully unlearned some time ago :-).